src/migrate/table-creator.js

```diff
@@ -10,21 +10,26 @@ export function ensureTable(tableName, schemaName, trxOrKnex) {
   const lockTableWithSchema = getLockTableNameWithSchema(tableName, schemaName);
   return getSchemaBuilder(trxOrKnex, schemaName)
     .hasTable(tableName)
-    .then(
-      (exists) =>
-        !exists && _createMigrationTable(tableName, schemaName, trxOrKnex)
-    )
-    .then(() => getSchemaBuilder(trxOrKnex, schemaName).hasTable(lockTable))
-    .then(
-      (exists) =>
+    .then((exists) => {
+      return !exists && _createMigrationTable(tableName, schemaName, trxOrKnex);
```

Comment:
it's a huge pain to debug arrow functions without {} and prettier really butchers formatting for them, hence the change